CONTRIBUTING.md

```diff
@@ -85,6 +84,20 @@ release from this repo, use the appropriate tag. New Relic usually pushes beta
 versions of a release to a working branch before tagging them for General
 Availability.
 
+## Dev Environment Setup
+
+1. Fork and clone the repo locally
+    - Fork the repository inside GitHub
+    - `git clone git@github.com:<gh username>/newrelic-ruby-agent.git`
+1. Pick a Ruby version
+    - Use rbenv or rvm to install a version of Ruby
+1. Install development dependencies
+    - `bundle install`
+1. Check that your env is setup correctly
+    - bundle exec rake
+
```

Comment:
Hello @ChaelCodes,  just a couple of things:
1) Please reword Dev on the title to Development
2) Thanks for adding the bulleted list of steps.  Could you remove the last sentence "If you have any issues setting up the repository, let us know in issues."  There is a link at the end for support.  Typically, when developers run into issues, they comment directly in their respective issues.